lib/alert/addon/components/alert-table/component.js

```diff
@@ -54,7 +57,30 @@ export default Component.extend({
   search: true,
   searchText: null,
 
+  clusterAlerts: function() {
+    const clusterId = get(this, 'clusterId');
+    const gs = get(this, 'globalStore');
+    return gs.all('clusterAlert').filterBy('clusterId', clusterId);
+  }.property('clusterId'),
```

Comment:
This needs to depend on `clusterAlert.@each.clusterId`, it's now not a live list and needs to be recomputed if a cluster comes or goes.  similar below.

Reply:
Thanks! PR Updated.

Reply:
You mean I need to change the code like the following? I don't know that clusterAlerts can depend on itself. 

```
  clusterAlerts: function() {
    const clusterId = get(this, 'clusterId');
    const gs = get(this, 'globalStore');
    return gs.all('clusterAlert').filterBy('clusterId', clusterId);
  }.property('clusterId', 'clusterAlerts.@each.{clusterId}'),
```